README.md

```diff
@@ -1,2 +1,32 @@
 # New_York_Times_Article
+
 A clone of a NYT article page
+
+![screenshot](./New_York_Times.png)
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://crappypapa.github.io/New_York_Times_Article/)
+
```

Comment:
- Add instructions for cloning, starting the app locally. Refer to [this](https://github.com/microverseinc/readme-template) template

Reply:
I have implemented the requested changes.